editor/components/toolbar/style.scss

```diff
@@ -30,6 +30,16 @@
 		background-color: $dark-gray-500;
 		color: $white;
 	}
+
+	&:after {
```

Comment:
Likely no difference since styles ensure there's no impact on layout when empty, but this should probably only be applied if the control actually has a `data-level` attribute:

```scss
&[data-level]:after {
```